templates/vm.conf.j2

```diff
@@ -0,0 +1,79 @@
+################################################################################
+# {{ ansible_managed }}
+################################################################################
+# {{ vm.host }} @{{ vm.org}}
+# {{ vm.description }}
+#-------------------------------------------------------------------------------
+# os:            {{ vm.os.type }} {{ vm.os.version }}
```

Comment:
Why all the infos here? They might change in the future, e.g. if I do a `do-release-upgrade` in my Ubuntu

Reply:
There should be a role which updates the vm to the selected ubuntu or debian release. And this file is updated automatically to the current selected release.

Reply:
Is there a role or should there be a role? Also, why all the infos? If I need the RAM, I just look into the host_vars